int32 face_id
int32 timestamp
Pose pose
ImageRect img_rect